# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocketChild.h
# Commit: b83124cf3a89
# Full Hash: b83124cf3a891091773bcfe2cd15e36155fc5a2d
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Teach WebrtcTCPSocket to use TLS when configured to. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45103
# ==============================================================================

diff -r b879a0044cf3 -r b83124cf3a89 media/mtransport/ipc/WebrtcTCPSocketChild.h
--- a/media/mtransport/ipc/WebrtcTCPSocketChild.h	Tue Sep 17 18:17:18 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocketChild.h	Tue Sep 17 18:17:24 2019 +0000
@@ -31,8 +31,8 @@
 
   void AsyncOpen(const nsCString& aHost, const int& aPort,
                  const nsCString& aLocalAddress, const int& aLocalPort,
-                 const net::LoadInfoArgs& aArgs, const nsCString& aAlpn,
-                 const dom::TabId& aTabId,
+                 bool aUseTls, const net::LoadInfoArgs& aArgs,
+                 const nsCString& aAlpn, const dom::TabId& aTabId,
                  NrSocketProxyConfig::ProxyPolicy aProxyPolicy);
 
   void AddIPDLReference() { AddRef(); }